torch/nn/modules/rnn.py

```diff
@@ -507,6 +507,23 @@ class LSTM(RNNBase):
         All the weights and biases are initialized from :math:`\mathcal{U}(-\sqrt{k}, \sqrt{k})`
         where :math:`k = \frac{1}{\text{hidden\_size}}`
 
+    .. warning::
+        There are known deterministic issues for LSTM using cuDNN 7.6.5, 8.0 on CUDA 10.1 or later.
+        If you encounter this and need deterministic behavior,
```

Comment:
encounter those

Reply:
Would a user actually know if they're seeing non-deterministic behavior? How would they reasonably (1) know to look for it and (2) find it?

It might be better to simply say, "You can enforce deterministic behavior by setting..."

Reply:
User may find non-deterministic behavior if they run the network multiple times, and eventually they file issue here. Yes, I think your language is better. I'll change that.